### PR TITLE
feat(oauth): add discord oauth

### PR DESCRIPTION
**Pull request by jzeuzs**
adds discord oauth :)

### CODE REVIEW

**Comment by jzeuzs** [comment]
ill mark this as draft for now, so we can support default discord avatars

File: packages/client-gql/src/user.gql [hunk 1]
```diff
@@ -21,6 +21,7 @@ query getUserByName($username: String!) {
     createdAt
     github
     twitter
+    discord
```
Comment (HenryLeC) [inline packages/client-gql/src/user.gql:25]: Nice job adding the discord token to getUserByName, can you also add the discord field to getCurrentUser please.

File: packages/api/src/modules/auth/index.ts [hunk 2]
```diff
@@ -2,13 +2,15 @@ import { Router } from 'express';
...
 import GitHubAuth from './providers/github';
 import TwitterAuth from './providers/twitter';
+import DiscordAuth from './providers/discord';
 
 export default (passport: PassportStatic): Router => {
   const authRouter = Router();
 
   /** Third party auth services */
   authRouter.use('/github', GitHubAuth(passport));
-  // authRouter.use('/twitter', TwitterAuth(passport));
```
Comment (HenryLeC) [inline packages/api/src/modules/auth/index.ts:11]: Well that's why Twitter OAuth wasn't working before lol

File: packages/client-gql/src/user.gql (hunk 1 above)
Comment (jzeuzs) [reply packages/client-gql/src/user.gql:25]: sure

File: packages/api/src/modules/auth/index.ts (hunk 2 above)
Comment (jzeuzs) [reply packages/api/src/modules/auth/index.ts:11]: lol

File: packages/client-gql/src/user.gql (hunk 1 above)
Comment (jzeuzs) [reply packages/client-gql/src/user.gql:25]: solved in `ff1d028`

**Review by HenryLeC — APPROVED** [review]

**Comment by HenryLeC** [comment]
@Angshu31 I think it looks good, you can merge into staging once your review

**Review by heybereket — APPROVED** [review]
Neat stuff, thanks for following the auth format :)

**Pull request merged**